README.md

```diff
@@ -425,12 +411,22 @@ The following configuration values can be used:
 
 ### `scriptId` (required)
 
-Specifies the id of the Google Script project that clasp will target. It is the part located inbetween `/d/` and `/edit` in your project's URL: `https://script.google.com/d/<SCRIPT_ID>/edit`.
+Specifies the id of the Google Script that clasp will target. It is the part located inbetween `/d/` and `/edit` in your script's URL: `https://script.google.com/d/<SCRIPT_ID>/edit`.
```

Comment:
https://developers.google.com/apps-script/reference/script/script-app#getScriptId()

The official document says `script project's unique ID`.
Therefore, I think this change is unnecessary.

Reply:
Correct. Please do not change this line.